Examples/Volume/VolumePicker/controlPanel.html

```diff
@@ -0,0 +1,28 @@
+<table>
+    <tr>
+        <td><b>Clip Plane 1</b></td>
+        <tr>        
+          <td>
+            Position
+            <input class='plane1Position' type="range" min="0" max="2.0" step="1" value="0" />
+          </td>
+          <td>
+            Rotation
+            <input class='plane1Rotation' type="range" min="0" max="2.0" step="1" value="0" />
+          </td>
+        </tr>
+    </tr>
+    <tr>
+        <td><b>Clip Plane 2</b></td>
+        <tr>        
+          <td>
+            Position
+            <input class='plane2Position' type="range" min="0" max="2.0" step="1" value="0" />
+          </td>
+          <td>
+            Rotation
+            <input class='plane2Rotation' type="range" min="0" max="2.0" step="1" value="0" />
+          </td>
+        </tr>
+    </tr>
```

Comment:
Can you please expose the threshold opacity as a configurable parameter because in the example it is not obvious that it works as intended:
![image](https://github.com/Kitware/vtk-js/assets/219044/d380475d-83f4-4c5c-a25e-3bde3313fbea)
(I guess a property to control the opacity transfer function could be useful to demonstrate that it is unrelated to the threshold of the picker)

Reply:
Is this you a looking for?
https://github.com/rodrigobasilio2022/vtk-js/blob/1f48bdc97ff6f73b4f137337b71604e7bcac45d9/Examples/Volume/VolumePicker/index.js#L188

Reply:
Yes, having a slider to choose such threshold would be nice.